.github/workflows/release.yml

```
@@ -0,0 +1,46 @@
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download OpenCode plugin
+        uses: actions/download-artifact@v4
+        with:
+          name: superego-opencode-plugin
+          path: ./artifacts
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)